Dockerfile

```diff
@@ -20,26 +20,11 @@ ENV GOFLAGS=-mod=vendor
 ENV CGO_ENABLED=0
 WORKDIR /src
 
+FROM moby/moby-bin:$DOCKER_VERSION AS docker-engine
+FROM dockereng/cli-bin:$DOCKER_VERSION AS docker-cli
```

Comment:
Do you think there's a need to call out that these are automated builds, not "official releases" (for the occasional reader)?


(We should also really fix those repositories to have a readme to outline their purpose / give the right expectations 😞 - let me see if someone can work on that)

Reply:
Added "for testing" comment on args if that sgty.